sass/pages/login/_login.scss

```diff
@@ -0,0 +1,108 @@
+body{
+	
+}
+
+section{
+	width: 100vh;
```

Comment:
You don't need to specify the height of the section, it should grow to fit the size of its inner elements, this is why you don't have to use the center mixin, when an element is absolute positioned the height disappears.